platform-ajax/src/main/java/com/softicar/platform/ajax/request/AjaxFileUploadListFetcher.java

```diff
@@ -0,0 +1,43 @@
+package com.softicar.platform.ajax.request;
+
+import com.softicar.platform.common.core.exceptions.SofticarIOException;
+import com.softicar.platform.dom.event.upload.IDomFileUpload;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Iterator;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.Part;
+
+class AjaxFileUploadListFetcher {
```

Comment:
see comments on `AjaxRequestMessageFetcher`... same here

Reply:
same here

Reply:
changed c'tor to public